### ISSUE TITLE
Polychromasia

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5956]
**Preferred term label:** Polychromasia

**Synonyms** Increased immature red blood cells; Increased immature erythrocytes

**Definition (free text, please give PubMed ID)** An abnormal increase of immature red blood cells in the bloodstream.

**Parent term (use hpo.jax.org/app)** Abnormal erythrocyte morphology HP:0001877

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think that this term is rarely used in human medicine and is largely synonymous with reticulocytosis (https://hpo.jax.org/app/browse/term/HP:0001923). I will add as a related synonym.
